videojs-contrib-dash.html

```diff
@@ -13,11 +13,10 @@
   <link href="css/jumbotron-narrow.css" rel="stylesheet">
 
   <!-- VideoJS, DashJS -->
-  <script src="js/video.js"></script>
-  <script src="js/dash.all.js"></script>
-  <script src="js/videojs-dash.min.js"></script>
-  <link rel="stylesheet"href="css/video-js.min.css" ></link>
-  <link rel="stylesheet" type="text/css" href="css/videojs-contrib-dash.css"></link>
+  <script src="//vjs.zencdn.net/5.9/video.js"></script>
```

Comment:
Now we're pulling this from the CDN, can you update the details here:

https://github.com/stuarthicks/stuarts-players/blob/615accb7055104bef3ae8ed9a534f22ec1e63f89/videojs-contrib-dash.html#L60